tools/devtools/wndt-bootstrap.mjs

```diff
@@ -0,0 +1,13 @@
+import {createWndtBanners, createWndtBlogPosts} from './utils.mjs';
```

Comment:
Not too sure, but you'll probably want to move this after the `config()` call to ensure things are really setup before doing anything else